src/webgl/camera.js

```diff
@@ -217,12 +217,12 @@ p5.RendererGL.prototype.perspective = function(fovy, aspect, near, far) {
     far = this.defaultCameraFar;
   }
 
-  this.cameraFOV = fovy;
+  this.cameraFOV = this._pInst._toRadians(fovy);
   this.cameraAspect = aspect;
   this.cameraNear = near;
   this.cameraFar = far;
 
-  this.uPMatrix = p5.Matrix.identity(this.pInst);
+  this.uPMatrix = p5.Matrix.identity();
```

Comment:
given other changes in this patch, if rotate is called on this later, won't the angle mode be lost?

Reply:
yes indeed!! but this is not a problem now since the _all_ the renderer's methods are _only_ passed radians, uPMatrix is never manipulated with degrees. so, in fact, we _do_ want the matrix to ignore `angleMode`.

Reply:
Ok, thank you. I need to run the tests again because additional changes were added in the newest commit.

Reply:
yeah, sorry about that. i had to merge again and fix the conflicts.